internal/proxy/impl.go

```diff
@@ -3080,15 +3068,20 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
 			queryShardPolicy:   roundRobinPolicy,
 		}
 
+		items := make([]zapcore.Field, 0)
+		items = append(items, zap.String("traceID", traceID))
+		items = append(items, zap.String("role", typeutil.ProxyRole))
+		items = append(items, zap.Int64("msgID", qt.Base.MsgID))
+		items = append(items, zap.Uint64("timestamp", qt.Base.Timestamp))
+		items = append(items, zap.String("db", queryRequest.DbName))
+		items = append(items, zap.String("collection", queryRequest.CollectionName))
+		items = append(items, zap.Any("partitions", queryRequest.PartitionNames))
+		items = append(items, zap.Any("OutputFields", queryRequest.OutputFields))
+
 		err := node.sched.dqQueue.Enqueue(qt)
 		if err != nil {
-			log.Debug("CalcDistance queryTask failed to enqueue",
-				zap.Error(err),
-				zap.String("traceID", traceID),
-				zap.String("role", typeutil.ProxyRole),
-				zap.String("db", queryRequest.DbName),
-				zap.String("collection", queryRequest.CollectionName),
-				zap.Any("partitions", queryRequest.PartitionNames))
+			items = append(items, zap.Error(err))
```

Comment:
Can just put in a single line:

log.Error("CalcDistance queryTask failed to enqueue", append(items, zap.Error(err))...)